<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>1371</width>
      <height>731</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="QueryInfo">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>651</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="ResultTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>1001</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;宋体&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>书号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>类别</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出版社</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出版年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>作者</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>单价</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>总藏书量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>库存数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>添加时间</string>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="selectSearcBookName_mood">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;宋体&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>精确</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>模糊</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="SearchModeSelect">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;宋体&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>类别</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>书名</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>出版社</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>年份</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>作者</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>价格</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="SearchinSelectedMode">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>10</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>911</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>图书类别</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出版社</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出版年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>作者</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>图书单价</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>总馆藏数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>库存数</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="AddRows">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DelRows">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>420</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TokenImportBooks">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>批量添加图书</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OktoAddBooks">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>440</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>确认添加</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLineEdit" name="BooklandCard_id">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>借书卡号：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BooklendCard_id_confirm">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="Books_lent_by_id">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>801</width>
        <height>381</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>选择</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>借出日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>经手人</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="Confirm_to_return_book">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>480</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>确认归还</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="Book_id_to_lend_confirm">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>书号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="BookId_to_lend">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="Book_in_id_tolend">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>1001</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 12pt &quot;宋体&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>选择</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>类别</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出版社</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出版年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>作者</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>价格</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>现有量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>添加时间</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="Confirm_to_lendbook">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>470</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>确认借阅</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lendCard_id_forlend">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>470</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>480</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>借书卡号：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>添加借书卡</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>删除借书卡</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="AddCards_table">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>621</width>
        <height>191</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>部门</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>类型</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="AddCards_confirm">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>250</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>确认添加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddCards_add">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddCards_del">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Card_id_todel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>370</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>370</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>卡号：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Confirm_to_del_card">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>430</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>确认删除</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Card_manage_Warning">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>370</y>
        <width>261</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6"/>
    <widget class="QWidget" name="page_7">
     <widget class="QLineEdit" name="LoginName">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="LoginPassword">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>密码:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Login_in_mode">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>330</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Login_info_Warning">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>130</y>
        <width>411</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QPushButton" name="Admin_add_account">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>添加账户</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Admin_del_account">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>380</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>删除账户</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="get_admin_accname">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="get_admin_name">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>240</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="get_admin_password">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="get_admin_contact">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>180</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="to_del_admin_accname">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>360</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="to_del_admin_password">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>430</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>360</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>账户名:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>430</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>密码:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>60</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>账户名:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>密码:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>联系方式:</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Acc_manage_Warning">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>60</y>
        <width>221</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="SearchBooks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>图书查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddBooks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>添加图书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LandCard">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>借书证管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReturnBooks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>还书管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LendBooks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>借书管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>管理员登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitLogin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Modify_Admins">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>修改管理员</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
